tilavarauspalvelu/models/equipment_category/model.py

```diff
@@ -11,8 +11,9 @@
 
 
 class EquipmentCategory(models.Model):
+    rank: int = models.PositiveIntegerField(default=0, null=False, blank=False, db_index=True)  # Used for ordering
```

Comment:
Perhaps the default should be a large number so that new categories are not automatically show at the top of results? You could find out the last value in the database, use a postgres sequence, or just use `qs.count()`.